src/goPackages.ts

```diff
@@ -75,17 +81,21 @@ function gopkgs(workDir?: string): Promise<Map<string, string>> {
 					}
 					const index = pkgPath.lastIndexOf('/');
 					const pkgName = index === -1 ? pkgPath : pkgPath.substr(index + 1);
-					pkgs.set(pkgPath, pkgName);
+					pkgs.set(pkgPath, {
+						name: pkgName,
+						isStd: pkgPath.includes(process.env['GOROOT'])
```

Comment:
This is the case of the old `gopkgs`. In that case, we will have to treat the output differently right?

Reply:
I am not entirely sure how the output of `{{.Dir}}` would look with an earlier version?

Reply:
https://github.com/tpng/gopkgs is the older tool. Can you give it a try?

Reply:
It most likely ignores the entire formatting input

Reply:
Oh sorry. I forgot to try this. Will do it tomorrow.